Source/data.f90

```diff
@@ -868,11 +868,11 @@ SUBROUTINE DEFINE_OUTPUT_QUANTITIES
 OUTPUT_QUANTITY(244)%SHORT_NAME = 'acc'
 
 OUTPUT_QUANTITY(245)%NAME       = 'HUMAN_CONTACT_LINEFORCE'
-OUTPUT_QUANTITY(245)%UNITS      = 'N/m'
+OUTPUT_QUANTITY(245)%UNITS      = 'N'
```

Comment:
The pressure term in 2D evacuation flow field is in unit of N/m
The force term in agent movement dynamics (Newton 2nd Law) is in unit of N. 

Please check the 'HUMAN_CONTACT_LINEFORCE' and 'HUMAN_TOTAL_LINEFORCE' in evac.f90
'HUMAN_CONTACT_LINEFORCE' -> HR%Sumforces -> CONTACT_F
'HUMAN_TOTAL_LINEFORCE' -> HR%Sumforces2 -> SOCIAL_F + CONTACT_F